drainer/pump_test.go

```diff
@@ -80,10 +80,12 @@ func (s *pumpSuite) TestPullBinlog(c *C) {
 
 	// ascending commitTs order
 	pullBinlogCommitTSChecker(commitTsArray, ret, binlogBytesChan, c)
+	time.Sleep(10 * time.Microsecond)
```

Comment:
Why sleep here?

Reply:
code: https://github.com/pingcap/tidb-binlog/blob/master/drainer/pump.go#L173
Because when `pullBinlogCommitTSChecker` is finished and quits, `p.latestTS` has not been updated yet.Because`binlogChanSize = 0`, `pullBinlogCommitTSChecker` send out the final binlog and quit but in pump it needs some time to update `p.latestTS`. So we sleep here to assure that `p.latestTS` has been updated before `assert`.